calico/datadog_checks/calico/check.py

```diff
@@ -0,0 +1,16 @@
+from datadog_checks.base import OpenMetricsBaseCheckV2
+
+from .metrics import METRIC_MAP
+
+
+class CalicoCheck(OpenMetricsBaseCheckV2):
```

Comment:
Just a FYI, since Calico uses OpenMetricsBaseCheckV2, it is python3-only compatible. This means that customers with Agent 7 can install Calico.